packages/trpc/server/routers/viewer/teams/getUpgradeable.handler.ts

```diff
@@ -11,7 +10,7 @@ type GetUpgradeableOptions = {
 };
 
 export const getUpgradeableHandler = async ({ ctx }: GetUpgradeableOptions) => {
-  if (!IS_TEAM_BILLING_ENABLED) return [];
+  // if (!IS_TEAM_BILLING_ENABLED) return [];
```

Comment:
same do we need to keep this ?

Reply:
I'll undo this. I was just testing this banner